shopfloor_packing_info/services/checkout.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Camptocamp SA (https://www.camptocamp.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+import os
+
+from odoo.addons.shopfloor.services.checkout import Checkout
+
+
+class CheckoutExt(Checkout):
+    _inherit = "base.shopfloor.process"
+
+    def _data_for_packing_info(self, picking):
+        res = super()._data_for_packing_info(picking)
+        if picking.picking_type_id.shopfloor_display_packing_info:
+            shopfloor_packing_info = (
+                picking.shopfloor_packing_info_id
+                and picking.shopfloor_packing_info_id.text
+                or ""
```

Comment:
I recommend to never use the `... and ... or ...` ternary "operator", because it can be very misleading: see the comment on https://docs.python.org/3.3/faq/programming.html#is-there-an-equivalent-of-c-s-ternary-operator (can't be an issue here but a good habit IMO)